content/diversity-accessibility.html

```diff
@@ -0,0 +1,92 @@
+type: page
+title: Diversity and accessibility
+slug: diversity-accessibility
+body_class_hack: talks
+---
+<h2>Diversity and accessibility at PyCon UK</h2>
+
+<p>Most of the global software industry is dominated by a very narrow demographic, of young, white, western men. This imbalance isn't good for anyone, and we want PyCon UK 2016 to be a milestone in the effort to address it.</p>
+
+<p>We will be making every effort to hold as inclusive an event as possible, and to shift the demographic of our audience and speakers. This is not just a vague aim, <a href="/news/target">it's a clear target</a>.</p>
```

Comment:
This link is broken, because we don't handle links to /news/xyz yet -- I can fix this.